ssh-u2f.py

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env python
```

Comment:
Yeah, I put myself in a situation where I was motivated to solve the problem because I'd already bought U2F tokens for my team :)